.github/workflows/release.yaml

```diff
@@ -31,9 +31,12 @@ jobs:
             -   name: Install Box
                 run: composer global require --dev humbug/box
 
+            -   name: Ensure the dependencies are up to date
+                run: make vendor
```

Comment:
this is purely to avoid noise. It can be that the `vendor` install does not have the right timestamp in which case calling `make build` will trigger another `composer install`. It is not a big deal, but it is noisy, so extracting this in a dedicate step helps to not pollute the important step bellow.